### PR TITLE
[WIP] multipath chapter 15.4 - further changes

### PR DESCRIPTION
**Pull request by mwilck**
### Further corrections and additions to the multipath chapter

#1229 is already very big. Creating this as a PR into the
mwilck/multipath-15.4 branch to show the differences.

### PR creator: Are there any relevant issues/feature requests?

* bsc#1179182
* jsc#SLE-19638

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [x] SLE 15 SP1
  - [x] SLE 15 SP0
- SLE 12
  - [x] SLE 12 SP5
  - [x] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

*Please do not merge: WIP*

### CODE REVIEW

File: xml/storage_multipath.xml [hunk 1]
```diff
@@ -447,6 +390,23 @@ To disable multipath support in <filename>initrd</filename>, run the command:
      the changes may require rescanning SCSI targets and reconfiguring
      multipathing on the host.
    </para>
+   <para>
+    If a disk array has more than one storage processor, ensure that the SAN
```
Comment (lvicoun) [inline xml/storage_multipath.xml:394]: Are there any arrays where there are more than 2 SP? I got confused by the "both storage processor" mentioned later. Usually there are 2 SP if I understand it correctly?

File: xml/storage_multipath.xml [hunk 2]
```diff
@@ -522,51 +482,89 @@ To disable multipath support in <filename>initrd</filename>, run the command:
...
-    in a production environment.
-   </para>
-   <para>
-    DM-MPIO protects against failures in the paths to the device, and not
-    failures in the device itself. If one of the active paths is lost (for
-    example, a network adapter breaks or a fiber-optic cable is removed), I/O
-    is redirected to the remaining paths. If all active paths fail,
-    inactive secondary paths must be woken up, so failover occurs with a delay
-    of up to 30 seconds, depending on the properties of the storage array.
+     The Device Mapper Multipath module takes care of the following
```
Comment (lvicoun) [inline xml/storage_multipath.xml:490]:
```suggestion
     The Device Mapper Multipath module handles the following
```

File: xml/storage_multipath.xml (hunk 1 above)
Comment (mwilck) [reply xml/storage_multipath.xml:395]: I can change "both" to "all" if you prefer.

**Comment by mwilck** [comment]
Rebased this PR onto the latest code base of #1229. The rebase was a bit of work, please let's continue here.

Also added some more commits to my rework.

File: xml/storage_multipath.xml (hunk 1 above)
Comment (lvicoun) [reply xml/storage_multipath.xml:372]: I'd delete this paragraph. IMHO it does not convey valuable information. Instead there could be a small paragraph about the purpose of storage processors?

Comment (mwilck) [reply xml/storage_multipath.xml]: Have you seen the version of this section in my last PR? I believe it's better understandable now.

Comment (lvicoun) [reply xml/storage_multipath.xml]: Yes, that looks good to me.

**Pull request merged**